src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -13,10 +14,11 @@ class SuggestionProvider extends React.Component {
     window.addEventListener('message', event => {
       if (event.origin !== intent.attributes.client) return null
       const { query, id } = event.data
-      this.provideSuggestions(query, id, intent)
+      // sometimes we get messages with undefined query & id, no idea why
+      if (query && id) {
+        this.provideSuggestions(query, id, intent)
+      }
```

Comment:
This is... concerning? Do you know what the messages are?

Reply:
Yeah, it's definitely not good, but I didn't want to investigate the cozy-bar at that time. The `intent` looks good, but `query` and `id` are undefined...